index.html

```diff
@@ -0,0 +1,243 @@
+
+
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="/Users/roobadi/Desktop/github/JS01Anaiya/STYLES/styles.css">
+</head>
+<body>
+  <h1>Harry Potter is cool!</h1>
+  <p>
```

Comment:
You have some interesting content here! Right now all of it seems to be inside one paragraph, so it might not be displaying as you expected. 

Suggest that you use separate paragraphs `<p></p>` for each group of text that you have.